### DISCUSSION TITLE
How can pass parameter on even @search

### DISCUSSION THREAD

**Discussion by mastrus** [sagalbot/vue-select#1255] [Q&A] [answered]
hi, I'm in v-for loop and I have several v-select element and on search I need to pass some parameters get from the active v-select tag.
Is there a way to do it?

**Comment by chrishubert** [comment, marked as answer]
Hello @mastrus ,

You can use the following to get it done:
```
<v-select @search="(search, loading) => mySearchFunction(search, loading, parameterToPassToMySearchFunction)"
          :options="[]"
          v-model="{}">
</v-select>
```

**Comment by deye9** [comment]
Thanks for this @chrishubert. You just saved me hours of googling and research.
